### PR TITLE
JAMES-1635 Increase embedded Cassandra startup timeout

### PR DESCRIPTION
**Pull request by chibenwa**
Increase embedded Cassandra startup timeout to the value configured in CassandraCluster used for unit tests.

This will avoid tests to fail on less effective computers.

### CODE REVIEW

File: backends-common/cassandra/src/test/java/org/apache/james/backends/cassandra/CassandraCluster.java [hunk 1]
```diff
@@ -42,10 +42,12 @@
 
     private static final long SLEEP_BEFORE_RETRY = 200;
     private static final int MAX_RETRY = 200;
+    private static final long EMBEDDED_CASSANDRA_TIMEOUT = MAX_RETRY * SLEEP_BEFORE_RETRY;
```
Comment (aduprat) [inline backends-common/cassandra/src/test/java/org/apache/james/backends/cassandra/CassandraCluster.java]: I would prefer an independent value, as the MAX_RETRY & SLEEP_BEFORE_RETRY don't make sense for this, IMHO.
You may also add the unit in the constant name.

Comment (chibenwa) [inline backends-common/cassandra/src/test/java/org/apache/james/backends/cassandra/CassandraCluster.java]: Ok for the unit.

I do not agree with you for the timeout independent value. I do not want Cassandra unit to throw before the expected timeout value expressed by the retry loop. I'm OK with it throwing after that. For me it make perfect sense.

In fact SLEEP_BEFORE_RETRY should also indicate its value

**Comment by aduprat** [comment]
:+1:

**Comment by chibenwa** [comment]
Committed

**Pull request closed without merging**